src/consts.ts

```diff
@@ -30,6 +30,8 @@ type RankInfoType = {
   color: string
   salary: number
 }
+const reusableH1 =
+  'font-unbounded text-primary flex-1 shrink-0 whitespace-nowrap text-xl font-semibold tracking-tight sm:grow-0'
```

Comment:
Why like this and not also similar to main in index.css

```css
h1 {
 @apply font-unbounded text-primary flex-1 shrink-0 whitespace-nowrap text-xl font-semibold tracking-tight sm:grow-0;
}
```

I don't see any benefit to doing this and just causes confusion what to use.

Reply:
Thats ok. Will fix in later pr